rpms/pulp-rpm/pulp-rpm.spec

```diff
@@ -0,0 +1,255 @@
+
+
+# ---- Admin Extensions --------------------------------------------------------
+%package admin-extensions
+Summary: The RPM admin client extensions
+Requires: pulp-admin-client = %{pulp_version}
+Requires: python-pulp-rpm-common = %{version}
+
+
+%description admin-extensions
+A collection of extensions that supplement and override generic admin
+client capabilities with RPM specific features.
+
+
+%files admin-extensions
+%license LICENSE
+%doc COPYRIGHT
+%{python_sitelib}/pulp_rpm/extensions/admin/
+%{python_sitelib}/pulp_rpm_extensions_admin*.egg-info
+
+
+# ---- Consumer Extensions -----------------------------------------------------
+%package consumer-extensions
+Summary: The RPM consumer client extensions
+Requires: pulp-consumer-client = %{pulp_version}
+Requires: python-pulp-rpm-common = %{version}
+
+
+%description consumer-extensions
+A collection of extensions that supplement and override generic consumer
+client capabilities with RPM specific features.
+
+
+%files consumer-extensions
+%license LICENSE
+%doc COPYRIGHT
+%{python_sitelib}/pulp_rpm/extensions/consumer/
+%{python_sitelib}/pulp_rpm_extensions_consumer*.egg-info
+
+
+# ---- Development package -----------------------------------------------------
+%package devel
+Summary: Pulp RPM devel package
+Requires: python-pulp-rpm-common = %{version}
+
+
+%description devel
+A collection of development utilities useful for developing pulp-rpm.
+
+
+%files devel
+%license LICENSE
+%doc COPYRIGHT
+%{python_sitelib}/pulp_rpm/devel/
+%{python_sitelib}/pulp_rpm_devel*.egg-info
+
+
+# ---- Documentation ----------------------------------------------------------
+%package doc 
+Summary: Pulp RPM documentation
+
+
+%description doc 
+Documentation for the Pulp RPM plugins.
+
+
+%files doc 
+%license LICENSE
+%doc AUTHORS COPYRIGHT
+%doc docs/_build/html/*
+
+
+# ---- Agent Handlers ----------------------------------------------------------
+%package handlers
+Summary: Pulp agent rpm handlers
+Requires: fedora-repos
+Requires: python-rhsm
+Requires: python-pulp-agent-lib = %{pulp_version}
+Requires: python-pulp-rpm-common = %{version}
+
+
+%description handlers
+A collection of handlers that provide both Linux and RPM specific
+functionality within the Pulp agent.  This includes RPM install, update,
+uninstall; RPM profile reporting; binding through yum repository
+management and Linux specific commands such as system reboot.
+
+
+%files handlers
+%license LICENSE
+%doc COPYRIGHT
+%{_sysconfdir}/pulp/agent/conf.d/bind.conf
+%{_sysconfdir}/pulp/agent/conf.d/linux.conf
+%{_sysconfdir}/pulp/agent/conf.d/rpm.conf
+%ghost %{_sysconfdir}/yum.repos.d/pulp.repo
+%{python_sitelib}/pulp_rpm/handlers/
+%{python_sitelib}/pulp_rpm_handlers*.egg-info
+
+
+# ---- Plugins -----------------------------------------------------------------
+%package plugins
+Summary: Pulp RPM plugins
+Requires: createrepo >= 0.9.9-21
+Requires: createrepo_c >= 0.4.1-1
```

Comment:
We really should drop the createrepo dependency. I think we use it in one place where we could use createrepo_c instead. It may be worth doing this before you bother to move createrepo into the SCL (if we end up doing the SCL at all)